### DISCUSSION TITLE
Invalid escape character in string

### DISCUSSION THREAD

**Discussion by brucegl** [huntabyte/shadcn-svelte#825] [Help] [answered]
Hi,

I made the change in src/lib/components/ui/.eslintrc as instructed to remove false positives, however VSCode displays `Invalid escape character in string` on the line `"varsIgnorePattern": "^\$\$(Props|Events|Slots|Generic)$"`. If I remove the backslashes, the issue goes away, but I'm not sure if I should be doing that?

Thanks

**Comment by shyakadavis** [comment, marked as answer]
Hello;

Trying using this rule instead - works fine for me, hope it does for you as well. 🙂 

```
varsIgnorePattern: '^(nameAttr|valueAttr|\\$\\$(Props|Events|Slots|Generic))$'
```

**Reply by brucegl** [reply]
Thanks @shyakadavis ,

Curious why you added nameAttr and valueAttr to the pattern?

Thanks

**Reply by shyakadavis** [reply]
That actually might be an oversight, sorry. This should also work just fine - I hope.

```
varsIgnorePattern: '^(\\$\\$(Props|Events|Slots|Generic))$'
```
